### PR TITLE
[13.0] [MIG] product_category_archive: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT27468

please @ernestotejeda @pedrobaeza review this

### CODE REVIEW

File: product_category_active/models/product.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        help="If unchecked, it will allow you to hide the "
+        "product category without removing it.",
+    )
+
+    @api.constrains("active")
+    def _check_archive(self):
+        to_archive = self.filtered(lambda r: not r.active)
+        if (
+            self.env["product.template"]
+            .with_context(active_test=False,)
```
Comment (victoralmau) [inline product_category_active/models/product.py]:
```suggestion
            .with_context(active_test=False)
```

**Review by victoralmau — APPROVED** [review]

File: product_category_active/i18n/product_category_archive.pot [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Translation of Odoo Server.
```
Comment (ernestotejeda) [inline product_category_active/i18n/product_category_archive.pot]: Remove i18n/product_category_**archive**.pot file

File: product_category_active/tests/test_product_category_archive.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
```
Comment (ernestotejeda) [inline product_category_active/tests/test_product_category_archive.py]: rename tests/test_product_category_**archive**.py to tests/test_product_category_**active**.py

File: product_category_active/tests/test_product_category_archive.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import SavepointCase
+
+
+class TestProductCategoryArchive(SavepointCase):
```
Comment (ernestotejeda) [inline product_category_active/tests/test_product_category_archive.py]:
```suggestion
class TestProductCategoryActive(SavepointCase):
```

**Comment by CarlosRoca13** [comment]
Changes done @victoralmau @ernestotejeda

**Review by ernestotejeda — APPROVED** [review]

**Comment by bodedra** [comment]
Can you make ci happy?

**Comment by CarlosRoca13** [comment]
For fixing the ci problems I make this PR https://github.com/OCA/product-attribute/pull/818

**Pull request merged**